ui/src/app/views/workspace/component.ts

```diff
@@ -323,22 +324,50 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.clusterService
         .localizeNotebook(this.workspace.namespace, this.workspace.id, fileList)
         .subscribe(() => {
-          this.alertCategory = 'alert-success';
-          this.alertMsg = 'File(s) have been saved';
-          this.showAlerts = true;
+          this.handleLocalizeSuccess();
           setTimeout(() => {
             this.resetAlerts();
           }, 5000);
         }, () => {
-          this.alertCategory = 'alert-danger';
-          this.alertMsg = 'There was an issue while saving file(s) please try again later';
-          this.showAlerts = true;
+          this.handleLocalizeError();
           setTimeout(() => {
             this.resetAlerts();
           }, 5000);
         });
   }
 
+  localizeAllFiles(): void {
+    this.errorHandlingService.retryApi(this.workspacesService
+        .retrieveAndLocalizeFiles(this.wsNamespace, this.wsId))
+        .subscribe(() => {
+              this.handleLocalizeSuccess();
+              setTimeout(() => {
+                this.resetAlerts();
+                this.clusterPulled = true;
```

Comment:
How is clusterPulled going to be used?

Reply:
ClusterPulled was an existing indicator to show the user the pop up  "Your notebook cluster is ready. Would you like to enter the cluster?"

So what i was going for was...give 3 seconds to show if or not the files have been saved and then display the pop up

Reply:
Got it. 

My hope is that 3 seconds would be enough to localize notebook files, but if there are really big notebooks (or perhaps other input files that get localized in future), 3 seconds might not be enough. We might want a bigger timeout.

Reply:
BTW, will the popup appear as soon as all the files are localized, or only after 3 seconds?

We should be able to make this pop up as soon as the files are localized (which normally would be much faster than 3 seconds.)

Reply:
3 seconds is just to display the success/error message as soon as the files have been localized.
So : localize files -> display message and wait for 3 sec-> clear the message and show the pop up saying cluster is ready

Reply:
What I'm asking is: if it only takes 1 second for the files to be localized, will the popup appear after 3 seconds or 1?

(It should be 1.)

Reply:
ahh ok yes 3. i can pull it out

Reply:
Fixed. clusterPulled will be true, i.e the cluster is ready to launch as soon as the files are localized